<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ROIAndVertex" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>612</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(75, 75, 75);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rectBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>矩形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ellipseBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>椭圆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lineBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>直线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polygonBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>多边形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pipeBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>管径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="angleBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>管角</string>
    </property>
   </widget>
   <widget class="ShadowLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>306</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="distanceBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>测距</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ROIAndVertex</class>
   <extends>QLabel</extends>
   <header>roiandvertex.h</header>
  </customwidget>
  <customwidget>
   <class>ShadowLabel</class>
   <extends>QLabel</extends>
   <header>shadowlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
